lib/rules/property-no-deprecated/README.md

```diff
@@ -0,0 +1,86 @@
+# property-no-deprecated
+
+Disallow deprecated properties.
+
+<!-- prettier-ignore -->
+```css
+  a { box-flex: 1; }
+/**      ↑
+ * Deprecated property */
+```
+
+This rule flags properties that were removed or deprecated after being in the CSS specifications, including editor drafts, and were subsequently either:
```

Comment:
question

You meant _previously_ instead of  _subsequently_?

Reply:
No, I meant "subsequently". I followed the formula used in the README files of [at-rule-no-deprecated](https://github.com/stylelint/stylelint/blob/main/lib/rules/at-rule-no-deprecated/README.md#L12) and [declaration-property-value-keyword-no-deprecated](https://github.com/stylelint/stylelint/blob/main/lib/rules/declaration-property-value-keyword-no-deprecated/README.md#L12) rules

Reply:
I thought it implied that the removal or deprecation happened first, followed by whichever condition applies.

Perhaps we can simply drop the indication of sequence?

> This rule flags properties that were removed or deprecated after being in the CSS specifications, including editor drafts, and were either: